### PR TITLE
Update statement and portfolio when transactions happen in app

### PR DESCRIPTION
**Pull request by qingweibinary**
Note: expiry does not work properly on statement, 
The problem is backend does not sell contract on expiry, the actual selling will be trigger by portfolio or statement call

to make it work on our app, we would need to have some daemon like function take consistently check for expiry and perform portfolio call, it's kind of short polling on client side  =(

@borisyankov any thoughts?
in addition, we would not fix this problem for virtual account, as backend do not want to waste resources on them, backend would only fix this for real account, probably by this month

Edit: Oh wait ~ it seems there's a way ... testing...

### CODE REVIEW

File: src/_actions/PortfolioActions.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const serverDataProposalOpenContract = serverResponse => ({
 
 export const updateNow = () => ({
     type: types.UPDATE_NOW,
+    now: Math.floor(Date.now() / 1000),
```
Comment (borisyankov) [inline src/_actions/PortfolioActions.js]: Should that be an utility function? nowAsEpoch()

File: src/_utils/ApiWorkaroundUtils.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as LiveData from '../_data/LiveData';
+
+export const forceStatementUpdate = () => {
+    const todayEpoch = Math.floor(Date.now() / 1000) - 86400;
```
Comment (borisyankov) [inline src/_utils/ApiWorkaroundUtils.js]: Looks like this also belongs to DateUtils

File: src/_actions/PortfolioActions.js (hunk 1 above)
Comment (qingweibinary) [inline src/_actions/PortfolioActions.js]: Yes, this should be

File: src/_utils/ApiWorkaroundUtils.js (hunk 2 above)
Comment (qingweibinary) [inline src/_utils/ApiWorkaroundUtils.js]: Not so sure, as it's difficult to name, it actually means the epoch on the starting point of today ~ 
Should I?

Comment (borisyankov) [inline src/_utils/ApiWorkaroundUtils.js]: Yep, serves as form of a 'comment'.

Comment (qingweibinary) [inline src/_utils/ApiWorkaroundUtils.js]: Fixed

File: src/_actions/PortfolioActions.js (hunk 1 above)
Comment (qingweibinary) [inline src/_actions/PortfolioActions.js]: Fixed

**Pull request merged**